spec/features/staff/users_can_view_an_organisation_as_xml_spec.rb

```diff
@@ -140,8 +140,9 @@
       end
 
       scenario "the XML file does not contain fund or programme activities in the organisation" do
-        _fund = create(:fund_activity, organisation: organisation)
-        _programme = create(:programme_activity, extending_organisation: organisation)
+        beis = create(:beis_organisation)
+        _fund = create(:fund_activity, organisation: beis)
+        _programme = create(:programme_activity, extending_organisation: beis)
```

Comment:
Shouldn't this have it's ending organisation set to `organisation` i.e. a delivery partner (as defined in the `let!` at the top of the file)..?